### PR TITLE
chore: bump deps

### PR DESCRIPTION
**Pull request by Arqu**

### CODE REVIEW

File: iroh-p2p/src/node.rs [hunk 1]
```diff
@@ -392,7 +392,7 @@ impl<KeyStorage: Storage> Node<KeyStorage> {
                             providers_so_far,
                             ..
                         })) => {
-                            if index.last {
+                            if step.last {
```
Comment (Arqu) [inline iroh-p2p/src/node.rs:395]: @dignifiedquire Hope I did this right

Comment (dignifiedquire) [reply iroh-p2p/src/node.rs:395]: yep, looks good

**Review by dignifiedquire — DISMISSED** [review]

File: stores/rocks/Cargo.toml [hunk 2]
```diff
@@ -9,7 +9,7 @@ description = "Implementation of RocksDB based Store"
 
 [dependencies]
 anyhow = "1.0.57"
-rocksdb = { git = "https://github.com/rust-rocksdb/rust-rocksdb", branch = "master" }
```
Comment (dignifiedquire) [inline stores/rocks/Cargo.toml:12]: we can switch to the latest release

File: iroh-store/Cargo.toml [hunk 3]
```diff
@@ -8,16 +8,16 @@ repository = "https://github.com/n0-computer/iroh"
 description = "Implementation of the storage part of iroh"
 
 [dependencies]
-rocksdb = { git = "https://github.com/rust-rocksdb/rust-rocksdb", branch = "master" }
+rocksdb = { git = "https://github.com/rust-rocksdb/rust-rocksdb", rev = "934855fe5429f515460cbac01d9e6417e04401bd" }
```
Comment (dignifiedquire) [inline iroh-store/Cargo.toml:11]: same here

**Review by dignifiedquire — APPROVED** [review]

File: iroh-store/src/store.rs [hunk 4]
```diff
@@ -137,6 +137,7 @@ impl Store {
                 let mut iter = db.full_iterator_cf(&cf_meta, IteratorMode::End);
                 let last_id = iter
                     .next()
+                    .and_then(|r| Some(r.unwrap()))
```
Comment (Arqu) [inline iroh-store/src/store.rs:140]: Not happy with this unwrap here. Any ideas for a better treatment?

Comment (dignifiedquire) [reply iroh-store/src/store.rs:140]: r.ok()

File: stores/rocks/src/fs.rs [hunk 5]
```diff
@@ -132,7 +132,8 @@ impl RocksFs {
 
     /// Deletes all elements in the database.
     pub fn clear(&self) -> Result<()> {
-        for (key, _) in self.db.full_iterator(rocksdb::IteratorMode::Start) {
+        for r in self.db.full_iterator(rocksdb::IteratorMode::Start) {
+            let (key, _) = r.unwrap();
```
Comment (dignifiedquire) [inline stores/rocks/src/fs.rs:136]: questionmark not unwrap

**Review by dignifiedquire — APPROVED** [review]

**Pull request merged**